packages/frontend/src/pages/Space.tsx

```diff
@@ -355,11 +353,7 @@ const Space: FC = () => {
                         [ColumnVisibility.SPACE]: false,
                     }}
                     enableBottomToolbar={false}
-                    initialAdminContentViewValue={
-                        userCanManageSpaceAndHasNoDirectAccessToSpace
-                            ? 'all'
-                            : 'shared'
-                    }
+                    enableRowSelection={userCanManageSpace}
```

Comment:
```suggestion
                    enableRowSelection={userCanManageSpace}
                    initialAdminContentViewValue={
                        userCanManageSpaceAndHasNoDirectAccessToSpace
                            ? 'all'
                            : 'shared'
                    }
```

seems like a restack error

Reply:
addressed in a newer version